cachi2/core/errors.py

```diff
@@ -62,6 +62,15 @@ def friendly_msg(self) -> str:
 class FetchError(Cachi2Error):
     """Cachi2 failed to fetch a dependency or other data needed to process a package."""
 
+    please_retry = (
+        "The error might be intermittent, please try again.\n"
+        "If the issue persists, please contact the maintainers."
```

Comment:
Same issue here, these "maintainers" could either be us or the admins of a system that uses Cachi2

Reply:
Re-phrased, telling users to contact us only if the error seems to be on our side.